tests/Api/Shop/ChannelTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Shop;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ChannelTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_gets_collection_of_channel(): void
```

Comment:
```suggestion
    public function it_gets_collection_of_channels(): void
```
but i am wondering if sth more precise will be better:
```suggestion
    public function it_gets_collection_with_single_default_channel(): void
```
or sth similar